src/app/components/harvest/components/metadata-review/folder-row.component.ts

```diff
@@ -165,16 +166,19 @@ export class FolderRowComponent {
   }
 
   public setSite(mapping: HarvestMapping, siteId: number): void {
+    if ( !isInstantiated(siteId) ) { return; }
     mapping.siteId = siteId;
     this.mappingsChange.emit();
   }
 
   public setOffset(mapping: HarvestMapping, offset: string): void {
+    if ( !isInstantiated(offset) ) { return; }
     mapping.utcOffset = offset;
     this.mappingsChange.emit();
   }
 
   public setIsRecursive(mapping: HarvestMapping, isRecursive: boolean): void {
+    if ( !isInstantiated(isRecursive) ) { return; }
```

Comment:
inconsistent formatting, and also why are we doing these bail outs? How does that fix the bug?

Reply:
I remember this fixing another issue when editing a harvest mapping, but this seems to been resolved when I changed the implementation of the harvestAPI